gc/verbose/schema.xsd

```diff
@@ -366,8 +366,6 @@ xmlns:vgc="http://www.ibm.com/j9/verbosegc">
 	<complexType name="gc-start">
 		<sequence maxOccurs="1" minOccurs="0">
 			<element ref="vgc:cpu-util" maxOccurs="1" minOccurs="0" />
-		</sequence>
-		<sequence maxOccurs="1" minOccurs="1">
 			<element ref="vgc:mem-info" maxOccurs="1" minOccurs="0" />
 		</sequence>
```

Comment:
while it's likely correct it's not as strong is it could be/it used to be
mem-info is mandatory in gc-start, but this now it relaxes it allowing it not to show up

Reply:
not clear how minOccurs between 'sequence' and 'element' gets combined, if it's 'or' or 'and' relationship 

perhaps, there are also correct variants (but also stronger), where minOccurs is 1 for either element mem-info or sequece or both

Reply:
> mem-info is mandatory in gc-start

Then `minOccurs` for that element should not have been zero; I can fix that here,
```
			<element ref="vgc:mem-info" maxOccurs="1" minOccurs="1" />
```
but that's really a separate concern. Let me know if you would like that change included here.

Reply:
Reading through https://www.w3.org/TR/xmlschema-1/#Complex_Type_Definitions some more, it seems attributes `minOccurs` and `maxOccurs` have no meaning on `<sequence>` elements. Perhaps they should be removed, but, again, I think that should be considered a separate concern.

Reply:
> > mem-info is mandatory in gc-start
> 
> Then `minOccurs` for that element should not have been zero; I can fix that here,
> 
> ```
> 			<element ref="vgc:mem-info" maxOccurs="1" minOccurs="1" />
> ```
> 
> but that's really a separate concern. Let me know if you would like that change included here.

I feel like it should be both there (while it was not there originally, what was incorrect), and in sequence (and it really was, but as you said per docs, perhaps not meaningful).

```
		<sequence maxOccurs="1" minOccurs="1">
 			<element ref="vgc:cpu-util" maxOccurs="1" minOccurs="0" />
 			<element ref="vgc:mem-info" maxOccurs="1" minOccurs="1" />
 		</sequence>
```


Then I would interpret this as from top down:

sequence 
```
cpu-util
mem-info 
```
can occur exactly once (cannot be repeated to 'cpu, mem, cpu, mem' and at least must show once)

but further because element cpu-util has minOccurs=0, than that 1 sequence can degenerate to

```
mem-info
```

If we just change minOccurs=1 for mem-info element, it still allows that the top level sequence does not show up , hence mem-info does not show up what is not correct.

But again, not sure if/how those attributes are interpreted when combined for elements and sequences.

I don't have a strong opinion. You can leave it even as-as, or adjust with mem-info minOccurs=1 or both sequence and min-info minOccur=1.

Reply:
I take back my comment that "attributes `minOccurs` and `maxOccurs` have no meaning on `<sequence>` elements"; I was reading about `<complexType>`, not `<sequence>`. Each `<sequence>` element in this file either omits minOccurs (or maxOccurs) or specifies the value `"1"` which is the default. I don't think we have any situation where we need to specify either, but I don't think they hurt anything by being present. (You'll notice that none of the samples in https://www.w3.org/TR/xmlschema-1/#Complex_Type_Definitions use either attribute.)

To summarize:
* `minOccurs` was wrong for `mem-info`
* `<sequence>` was illegally repeated

I'll update this to correct both.

Reply:
I also corrected `minOccurs` for `<mem-info>` within `<gc-end>`.

Reply:
> I also corrected `minOccurs` for `<mem-info>` within `<gc-end>`.

yes, it's mandatory there, too